### ISSUE TITLE
4.06 support

### ISSUE THREAD

**Issue by copy** [janestreet/ppx_core#12]
Installing in a `4.06.0+rc1+default-unsafe-string` switch yields:

```
#=== ERROR while installing ppx_core.v0.9.0 ===================================#
# opam-version 1.2.2
# os           linux
# command      jbuilder build --only-packages ppx_core --root . -j 4 @install
# path         /home/fabian/.opam/4.06.0+rc1+default-unsafe-string/build/ppx_core.v0.9.0
# compiler     4.06.0+rc1+default-unsafe-string
# exit-code    1
# env-file     /home/fabian/.opam/4.06.0+rc1+default-unsafe-string/build/ppx_core.v0.9.0/ppx_core-16801-5f0cd0.env
# stdout-file  /home/fabian/.opam/4.06.0+rc1+default-unsafe-string/build/ppx_core.v0.9.0/ppx_core-16801-5f0cd0.out
# stderr-file  /home/fabian/.opam/4.06.0+rc1+default-unsafe-string/build/ppx_core.v0.9.0/ppx_core-16801-5f0cd0.err
### stderr ###
# Error: The implementation src/location.ml
# [...]
#        ...
#        In module Error:
#        Values do not match:
#          val of_exn : exn -> [ `Already_displayed | `Ok of L.error ] option
#        is not included in
#          val of_exn : Ppx_core__.Import.exn -> t Ppx_core__.Import.option
#        File "src/location.mli", line 51, characters 2-30:
#          Expected declaration
#        File "src/location.ml", line 57, characters 6-12: Actual declaration
```

**Comment by trefis** [comment]
We are currently working on a fix and hope to release a 4.06-compatible version on opam in the next few days.

Sorry, and thanks for the report!

**Comment by copy** [comment]
I can confirm ppx_core v0.9.2 working in 4.06.0+default-unsafe-string, thanks!
